### DISCUSSION TITLE
Atan2 node please

### DISCUSSION THREAD

**Discussion by hitrpr** [bnpr/Malt#448] [Feedback & Feature Requests]
This node/procedure is usually used to get angle from vector/position.
For radial/spherical/cylindrical coordinates etc. All about rotation around the point.

here how it looks now
![изображение](https://user-images.githubusercontent.com/21099918/202252208-cd63a0a8-10b8-441b-9198-4661e017954d.png)

Blend file: [spiral.zip](https://github.com/bnpr/Malt/files/10024253/spiral.zip)

And it is impossible to make own group with atan2 now.

reference https://en.wikipedia.org/wiki/Atan2

**Comment by pragma37** [comment]
Done! https://github.com/bnpr/Malt/commit/31426b84f4ff5809386ac39cd3925393893f5a28 👍
